pulp_python/app/tasks/publish.py

```diff
@@ -149,4 +131,27 @@ def find_artifact():
             publication=publication,
             file=File(open(metadata_relative_path, 'rb'))
         )
-        project_metadata.save()
+        project_metadata.save()  # change to bulk create when multi-table supported
+
+    ind = 0
+    current_name = index_names[ind][0]
+    package_releases = []
+    published_artifacts = []  # This list can grow up to 4 million elements for full PyPI, can be cleared if we do multiple bulk_creates
```

Comment:
This list can also grow up to 4 million elements, but it can be cleared if I bulk_create after every N releases.